packages/block-library/src/latest-comments/edit.js

```diff
@@ -75,6 +82,7 @@ export default function LatestComments( { attributes, setAttributes } ) {
 				<ServerSideRender
 					block="core/latest-comments"
 					attributes={ attributes }
+					urlQueryArgs={ { _locale: siteLanguage } }
```

Comment:
Just wondering whether we need to even grab the site language? If we set this to an empty string, would that be enough to bypass the middleware that sets the locale to `user` for the request?

In core, it looks like [once the request is processed](https://github.com/WordPress/WordPress/blob/HEAD/wp-includes/l10n.php#L137), if the `_locale` is an empty string, it should fall back to the site's locale?

Reply:
Yeah that's a really good point. It also fixes a bug I've discovered where sometimes when I refresh the page I see two calls to the block render API: first with `_locale=user` then immediately followed with `_locale=sitelang`. I'm guessing the site language isn't immediately available in the store so `useSelect()` is returning `siteLanguage=undefined` for the initial render.

If I set `_locale=''` then that's enough to bypass the [user locale middleware](https://github.com/WordPress/gutenberg/blob/trunk/packages/api-fetch/src/middlewares/user-locale.js#L12) which is adding the `_locale='user'` string by default. But it does still send a `?_locale=` as part of the request. It'd be tidy if there was a way to disable the user locale middleware entirely, but I'll ask about that in `#core-editor`

Reply:
>  It'd be tidy if there was a way to disable the user locale middleware entirely, but I'll ask about that in #core-editor

Probably not a _huge_ deal to be sending an empty query param, but that sounds good, it'd be ideal if there was a clean way to disable it when needed 👍